src/internal/clusterstate/v2.8.0/clusterstate.go

```diff
@@ -1,7 +1,11 @@
 package v2_8_0
 
 import (
+	"context"
+
+	col "github.com/pachyderm/pachyderm/v2/src/internal/collection"
```

Comment:
we should avoid importing collection + ppsdb packaged in case they change. In each of the migrations since 2.4, we've tried to pull more of this into the corresponding clusterstate

Reply:
You’re correct, and I should have remembered that. Refactoring …